user-interface/frontend/themes/datamanager/components/vaadin-custom.css

```
@@ -40,6 +40,11 @@ vaadin-multi-select-combo-box::part(toggle-button)::before {
   color: var(--lumo-primary-color);
 }
 
+/* Hide toggle button */
+vaadin-multi-select-combo-box.no-chevron::part(toggle-button)::before {
```

```suggestion
vaadin-multi-select-combo-box.no-chevron::part(toggle-button) {
```
Why only the `::before` part?